act-rules-format.bs

```diff
@@ -234,44 +304,40 @@ When a single URL can be used to reference the web page, or other test subject,
 
 Outcome {#output-outcome}
-------------------
+-------------------------
 
-The procedure of a rule MUST always return one of the following outcomes:
+The definition of a rule MUST always result in one of the following outcomes:
 
-- **Passed**: All [expectations](#expectations) for the [Test Target](#output-test-target) were true
+- **Passed**: All [expectations](#test-expectations) for the [Test Target](#output-test-target) were true
 - **Failed**: One or more expectations for the Test Target was false
 - **Inapplicable**: There were no Test Targets in the [Test Subject](#output-test-subject)
 
 <div class="note">
   While *inapplicable* is a valid result for ACT Rules, it may not be a valid result for all [accessibility requirements](#structure-accessibility-requirements). Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to true (passed) or false (failed). This translation of results is part of [output aggregation](#output-aggregation)
 </div>
 
-Ensure Comparable Results {#output-comparable}
--------------------------
-
-In addition to the [Test Target](#output-test-target) and the [Outcome](#output-outcome), ACT Rule output data MUST include the following contextual information:
-
-- the Web page, file or other [test subject](#output-test-subject) the rule was applied to and
-- an [identifier](#unique-identifier) of the rule itself.
+<div class="note">
+  In addition to `Passed` `Failed` and `Inapplicable`, [[EARL10-Schema]] also defined an `Incomplete` outcome. While this should never be the outcome of a rule when applied in its entirety, it often happens that rules are only partially executed. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the "Incomplete" outcome.
+</div>
 
 
 Rule Quality Assurance {#quality}
 =================================
 
-Implementation Validation {#quality-implement}
-----------------------------------------------
+Test Cases (Atomic rules only) {#quality-test-cases}
+----------------------------------------------------
 
-Implementation tests are tests for accessibility test tools. These tests enable the developers and users of ATTs to validate the implementation of an ACT Rule. An ACT rule MUST have implementation tests for the [applicability](#test-applicability), as well as for each [expectation](#expectations) in the rule.
+Test cases are (snippets of) content that can be used to validate the implementation of an [atomic rule](#atomic-rule-outline). They consist of two pieces of data, a snippet of each [test aspect](#input-aspects) for a rule, and the an [expected result](#output) that should come from that rule. Test cases serve two functions, firstly as example scenarios for readers to understand when a rule passes, when it fails, and when it is inapplicable. But also for developers and users of automated accessibility test tools to validate that a rule is correctly implemented.
 
```

Comment:
editorial 2nd sentence: "...and the [expected result]..."  - remove "an"
The last sentence is really part of the previous sentence.  Perhaps changing to a bulleted list with 2 bullets will help make this better.